packages/shared_preferences/ios/Classes/SharedPreferencesPlugin.m

```diff
@@ -20,6 +20,10 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
     } else if ([method isEqualToString:@"setBool"]) {
       NSString *key = arguments[@"key"];
       NSNumber *value = arguments[@"value"];
+      if ([[NSNull null] isEqual:value])
+        [[NSUserDefaults standardUserDefaults] removeObjectForKey:key];
+      else
+        [[NSUserDefaults standardUserDefaults] setValue:value forKey:key];
```

Comment:
Should this be `setBool` instead of `setValue`?

Reply:
I had meant to use the line below instead